notebook/converted/benchmark_results.py

```diff
@@ -1450,8 +1450,66 @@ def return_unlabeled():
 plot
 
 
-# In[ ]:
+# ## Tissue Split
+
+# In[2]:
+
+
+in_files = glob.glob('../../results/tissue-split.*.tsv')
+print(in_files[:5])
+
+
+# In[3]:
+
 
+tissue_metrics = pd.DataFrame()
 
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('biobert.')[-1]
+    model_info = model_info.split('.')
+    model_info = model_info[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+            
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
 
+tissue_metrics
+
+
+# In[4]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='is_pretrained')) 
```

Comment:
I'm surprised that there's not more heterogeneity/variance between splits. I'd assume that transfer between some (related) tissues would work well, but other tissues would be too different to share much information. But it looks here like none of your train/test splits really work well (which I guess is plausible, but surprising at least to me).

Reply:
There are actually only two tissue splits (mutiplying model_types * pretrain_or_random * cv_splits * initializations * data subsets by another factor would make the experiment take even longer). I'd expect that there are splits that would perform better, that may be an interesting future experiment (e.g. pretraining based on tissue similarity).

Reply:
Makes sense!